.github/workflows/ci.yml

```diff
@@ -0,0 +1,82 @@
+name: CI
+on:
+  # Trigger analysis when pushing to your main branches, and when creating a pull request.
+  push:
+    branches:
+      - main
+      - master
+      - develop
+      - 'releases/**'
+  pull_request:
+      types: [opened, synchronize, reopened]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          cache: 'yarn'
+      - run: yarn
```

Comment:
Acá qué opinás, es mejor un `yarn install` común (o `yarn` que es lo mismo) o un `yarn install --frozen-lockfile`?

Yo había puesto `--frozen-lockfile` que es equivalente a `npm ci`, y es lo que recomiendan para entornos automatizados como este ([este ejemplo de la docu](https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs) usa `npm ci`, y en [esta respuesta de SO](https://stackoverflow.com/a/53325242) habla un poco de las diferencias). Básicamente valida que el `yarn.lock` contenga lo necesario y suficiente para las dependencias listadas del `package.json` y tira error si no, garantizando una instalación "limpia".

Reply:
Pense que habiamos pasado a `npm` ya. Creo que deberiamos empezar a usar `npm`, la version 1 de `yarn` quedo super vieja y desactualizada y ni tiene mantenimiento.

Tambien se podria actualizar yarn, pero creo que una vez se probo y aparecieron issues o cosas que no sabiamos como manejar. `npm` tambien evoluciono mucho creo durante todo este tiempo que venimos usando `yarn`, asi que la opcion facil seria pasar a `npm`, y despues analizar cual conviene mas.

Reply:
Muchas cosas que comentar jaja:
1. Frozen-lockfile: Lo saqué por la forma en que se corren el instalador, tiene que correr para 4 proyectos, el base, los dos de ejemplo y el de las docs. Voy a ver de hacerlo más granular y a mano, no con el script de bootstrap como se hace ahora.
2. Re biche, yo no me pude a investigarlo mucho y casi todo lo seguimos haciendo con yarn porque en su momento manejaba bastante mejor las dependencias. En este caso diría de migrar todos los paquetes y el template a npm pero vamos a tener que planificarlo y hacerlo de a poco.

Reply:
Ahh de una de una. Respecto a pasar a NPM, nunca me metí en tema pero dadas esas circunstancias banco la migración